uart.py

```diff
@@ -0,0 +1,18 @@
+import serial
+
+UART_0 = '/dev/ttyS0'
+UART_1 = '/dev/ttyAMA0'
+
+class UART():
+    def __init__(self, port, baud):
+        self.ser = serial.Serial(port, baud)
```

Comment:
can we give this a better name than `ser`? any reason we can't just call it `serial`?

Reply:
its an extra 3 chars to type :)